css-three-column-layout/styles.css

```diff
@@ -0,0 +1,45 @@
+body {
+  background-color: #1a96d9;
+}
+
+* {
+  justify-content: center;
+  margin: 3%;
+}
```

Comment:
You want to avoid giving properties like this to the universal selector. `justify-content` will only be applied to elements with `display: flex`, so we don't want to apply this to every single element, that should only get applied to elements with `display: flex`.

Same goes for `margin`, we don't want to give every single element a `margin`, we only want to give `margin` to specific elements that need it.